scripts/dialog_text2code.py

```diff
@@ -36,9 +35,11 @@
     parser.add_argument('-i', '--inplace', required=False, help='replace input dialog by output.', action='store_true')
     parser.add_argument('-s', '--soft', required=False, help='soft name policy - change intents and entities names without error.', action='store_true', default="")
     parser.add_argument('-v', '--verbose', required=False, help='verbosity', action='store_true')
+    parser.add_argument('--log', type=str.upper, default=None, choices=list(logging._levelToName.values()))
     args = parser.parse_args(sys.argv[1:])
+    
+    setLoggerConfig(args.log, args.verbose)
```

Comment:
I know that it is not part of this issue and it is documented, but if we do not change all scripts missing `main` function now, we risk that when somebody does that he omits to put here `if __name__ == '__main__':` (it will be very difficult to find the source of problem because we do not have tests for logging and it could happen that we will realize that something in logs missing after sometime).

Reply:
Fixed in 605deecbe1b1b558adec61b8c8eeea4070e6e5c9.